trunk/examples/programs/regression/c/ACSL-ghostUpdateGlobal.c

```diff
@@ -0,0 +1,12 @@
+//#Safe
+// Author: schuessf@informatik.uni-freiburg.de
+// Date: 2023-09-18
+
+//@ ghost int i;
+
+int main() {
+  int x = 0;
+  //@ ghost i = x;
```

Comment:
In the PR comment you write

> `//@ ghost x = expr;` (local only)

But here it seems you use it with a global ghost variable?